code/pom.xml

```diff
@@ -110,6 +152,32 @@
 
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
```

Comment:
use build-helper-plugin instead. The difference is that it adds those legal files also to source jar

Reply:
Fixed.